From a87278bddd432ca104159fb2e9b1bea5f866dbb2 Mon Sep 17 00:00:00 2001
From: Moritz Angermann <moritz.angermann@gmail.com>
Date: Wed, 17 Feb 2021 21:13:07 +0800
Subject: [PATCH] [configure] Fix _AC_PROG_CC_99

_AC_PROG_CC_99 isn't available anymore in recent autoconf versions.
This broke on aarch64-darwin.
---
 aclocal.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/aclocal.m4 b/aclocal.m4
index 73791cd1b3..9911775295 100644
--- a/aclocal.m4
+++ b/aclocal.m4
@@ -659,7 +659,7 @@ AC_DEFUN([FP_SET_CFLAGS_C99],
     CPPFLAGS="$$3"
     unset ac_cv_prog_cc_c99
     dnl perform detection
-    _AC_PROG_CC_C99
+    AC_PROG_CC_C99
     fp_cc_c99="$ac_cv_prog_cc_c99"
     case "x$ac_cv_prog_cc_c99" in
       x)   ;; # noop
-- 
2.30.0

